samza-test/src/main/java/org/apache/samza/test/framework/StateAssert.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.samza.test.framework;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.samza.storage.kv.RocksDbKeyValueReader;
+import org.apache.samza.storage.kv.RocksDbTableDescriptor;
+
+import static org.junit.Assert.*;
+
+/**
+ *  Assertion utils on the content of a Table described by
+ *  {@link org.apache.samza.operators.TableDescriptor}.
+ */
+public class StateAssert {
+
+  /**
+   * Verifies that the {@code key} is present in the table described by {@code RocksDbTableDescriptor}
+   * @param key expected key in the table
+   * @param tableDescriptor describes the RocksDb table
+   */
+  public static void contains(Object key, RocksDbTableDescriptor tableDescriptor) {
+    boolean contains = false;
+    for (File partition: getPartitionDirectories(tableDescriptor.getTableId())) {
+      RocksDbKeyValueReader reader = new RocksDbKeyValueReader(tableDescriptor, partition.getAbsolutePath());
```

Comment:
SamzaContainer has multiple tasks and each task has a associated state-store.

Two different tasks(T1, T2) running on a single container will have state-store created under different directories viz `{RocksDBBaseDir}/T1/storeName`, `{RocksDBBaseDir}/T2/storeName` . 

In the above case, when the user invokes `get`, `contains`  API, it's not clear from which task state-store the user is reading the key from and what are our `StreamAssert` guarantees?